Beyond06-SOW.md

```diff
@@ -0,0 +1,344 @@
+    3. Store or modify information in the database as a response to user input
+
+2. Implement automated tests for the features you've completed
+
+Demo contents
+
+Your demo presentation should include the following elements:
+
+1. One or more video segments showing how the features you've implemented look and work from a user's point of view.
+2. An explanation of how you've tested the features you've developed. You should include the following details:
+
+    1. Which manual tests were carried out. How does one decide if a test was successful?
+    2. Which automated tests were written
+    3. Are failure scenarios being tested? In which way?
+    4. What aspects of the features you've written are not being tested?
+3. A demonstration of the application tests being run in GitHub actions.
+
+### Demo 4 (Jan 2): Final Project Demo
+
+The purpose of this demo is to show your development process and should includes:
+
+1. Project background - Explain in general the motivation and the requirements.
+2. Architecture -adding a chart that describes all system components.
+3. Results (features)
+4. Demo - video that presents your full system (UI, DB, APIs, CI)
+5. Development processing - share the process you made
+6. Development challenges
+7. What did you learn
+
+## Division into project groups (till Oct22th)
+
+After the first 2 weeks you'll be divided into 4 groups.
+
+Each group will have mentors:
+
+* group1: Yariv, Arie, Shimrit & Omer
+* group2: Kobi, Michal, Nati & Nahshon
+* group3: Barak, Daniel, Guy & Eran
+
+## Project ideas
+
+* Platform for freelancers to offer their services
+* P2P Lending service
+* Dating website
+* Drop shipping web store - selling other people's products on your website
+* Platform for finding part time jobs for students
+
+## Syllabus (list of sessions)
+
+| Session number | Date                   | Topic(s)                                                                                                                                                                                                                                                     | Lecturer             |
+|----------------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|                                                                                                     | 1              | 10.10                  | Intro session (housekeeping) ~50 min +Talk about initial challenges and student selection                                                                                                                                                                    | Liora Milbaum        |
+|                |                        | Self learning: Tool introduction challenge
```

Comment:
I think mentioning the 1st challenge both here and in the submission data might be confusing - lets either stick to how I wrote it in the spreadsheet (hand-off v.s. submission) or specify only the submission dates.

Reply:
Not seeing any change in the file about this